src/org/zaproxy/zap/extension/custompayloads/ZapAddOn.xml

```diff
@@ -0,0 +1,17 @@
+<zapaddon>
+	<name>Custom Payloads</name>
+	<version>1</version>
```

Comment:
This manifest seems to be missing a status element, like:
https://github.com/zaproxy/zap-extensions/blob/08b79419e4bd8a71f5eb019a18f6063b593b7243/src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml#L4